UI/css/pageNotFound.css

```diff
@@ -0,0 +1,27 @@
+#icon {
+    display: block;
+    height: auto;
+    margin: 0 auto;
+    max-width: 400px;
+    max-height: 400px;
+    padding-bottom: 50px;
+    padding-top: 50px;
+    width: auto;
+}
+
+#main-card {
+    max-width: 600px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: auto;
+    margin-bottom: auto;
+    text-align: center;
+}
+
+#card-container {
+    background-color: lightgray;
+}
+
+body {
```

Comment:
let's change this into a class

Reply:
What do you mean?

Reply:
so this is affecting the body, you can change this into like:
`.notFoundBody` and add that to the `.pug` file

Reply:
Ah, so the body itself isn't directly used, but with the same effect either way. Alright